src/main.rs

```diff
@@ -152,6 +152,13 @@ fn main() {
                 .value_name("FILE")
                 .help("Export the timing results to the given file in JSON format."),
         )
+        .arg(
+            Arg::with_name("export-md")
```

Comment:
I prefer descriptive long names for the `--long-options`. Can we use `--export-markdown` here? I don't think it's something that users will type very often.

Reply:
Sure